ldk/javascript/src/webpack/check-loop-open-handler-plugin.ts

```diff
@@ -0,0 +1,29 @@
+import { Compiler, WebpackError } from 'webpack';
+
+const assetName = 'loop.js';
+const warningMsg =
+  'Please provide a way for users to open your Loop. You can do this using the ui.loopOpenHandler. This is required to submit your Loop.';
+const plugin = 'CheckLoopOpenHandlerPlugin';
+export class CheckLoopOpenHandlerPlugin {
+  apply(compiler: Compiler): void {
+    compiler.hooks.thisCompilation.tap(plugin, (compilation) => {
+      compilation.hooks.processAssets.tap({ name: plugin }, (assets) => {
+        const loopjs = assets[assetName];
+        if (loopjs) {
+          const fileContent = loopjs.source();
+          if (
+            !fileContent?.includes('ui.loopOpenHandler(') ||
+            fileContent?.includes('// ui.loopOpenHandler(') ||
+            fileContent?.includes('// ui.loopOpenHandler(') ||
```

Comment:
This is the same as the previous line (unless I'm missing something)

Reply:
you are right, i meant it to be `fileContent?.includes('//ui.loopOpenHandler(')`. ill update

Reply:
Gotcha, thanks for clarifying